Gruntfile.js

```diff
@@ -53,7 +53,18 @@ module.exports = function(grunt) {
           // goog.provide and goog.require statements are added.
           shouldProvideRequireSoyNamespaces: false
         }
+      },
+      py: {
+         src: ['templates', 'soy', '**', '*.soy'].join('/'),
+        soyToJsJarPath: [process.env.HOME, 'bin', 'google_closure_templates',
+                         'SoyToPySrcCompiler.jar'].join('/'),
+        outputPathFormat: [targetDirectory, 'generated', '{INPUT_FILE_NAME_NO_EXT}.py'].join('/'),
+        options: {
+          // This points to the package - targetDirectory/generated/soyPython
```

Comment:
The python files from the closure-templates/python directory will be moved to out/generated/soyPython by the copy task, so it only needs to point to the directory SoyPython